config_test.go

```diff
@@ -0,0 +1,128 @@
+package memberlist
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+
+	multierror "github.com/hashicorp/go-multierror"
+)
+
+func Test_IsValidAddressDefaults(t *testing.T) {
+	tests := []string{
+		"127.0.0.1",
+		"127.0.0.5",
+		"10.0.0.9",
+		"172.16.0.7",
+		"192.168.2.1",
+		"fe80::aede:48ff:fe00:1122",
+		"::1",
+	}
+	config := DefaultLANConfig()
+	for _, ip := range tests {
+		localV4 := net.ParseIP(ip)
+		if err := config.IPAllowed(localV4); err != nil {
+			t.Fatalf("IP %s Localhost Should be accepted for LAN", ip)
+		}
+	}
+	config = DefaultWANConfig()
+	for _, ip := range tests {
+		localV4 := net.ParseIP(ip)
+		if err := config.IPAllowed(localV4); err != nil {
+			t.Fatalf("IP %s Localhost Should be accepted for WAN", ip)
+		}
+	}
+}
+
+// parseCIDRs return a possible empty list of all Network that have been parsed
+// In case of error, it returns successfully parsed CIDRs and the last error found
+// If nil is given it returns nil, nil
+func parseCIDRs(v []string) ([]net.IPNet, error) {
```

Comment:
Which one of `parseCIDRs` and `ParseCIDRs` do you want to keep?

Reply:
I kept the public one since it might be useful for the program using the lib